### ISSUE TITLE
Teleportation

### ISSUE THREAD

**Issue by Shaac** [prologin/prologin2014#12]
> ###### Originally posted 2014-05-07 15:12 by [Shaac](https://bitbucket.org/Shaac/) on Bitbucket

What is happening in the dump in turn 14 for player 4?
It is not intended but mostly it should be not possible: some unit here move faster than they should.

**Comment by ejls** [comment]
> ###### Originally posted 2014-05-07 17:37 by [ejls](https://bitbucket.org/ejls/) on Bitbucket

case:

```
#!

creer(1)
deplacer((0,0), (0,4), 1)
findetour

creer(1)
deplacer((0,0), (0,4), 1)
deplacer((0,4), (0,8), 1)
deplacer((0,4), (0,8), 1)
```
Il y a maintenant 2 unites en (0, 8) alors que le dernier deplacer devrait retourner "pas d'unite".

**Comment by ejls** [comment]
> ###### Originally posted 2014-05-07 17:44 by [ejls](https://bitbucket.org/ejls/) on Bitbucket

ci-joint un code c++ qui reproduit l'erreur.

**Comment by bakablue** [comment]
> ###### Originally posted 2014-05-08 01:03 by [bakablue](https://bitbucket.org/bakablue/) on Bitbucket

fix getters of nb_wizards(_movable) and copy constructor in cell fixes #12


→ <<cset 8e0c07a7d7f6>>

**Comment by pmderodat** [comment]
> ###### Originally posted 2014-05-11 11:43 by [pmderodat](https://bitbucket.org/pmderodat/) on Bitbucket

fix getters of nb_wizards(_movable) and copy constructor in cell fixes #12


→ <<cset 5b938394ffdd>>
